deploy-tool/main.go

```diff
@@ -193,13 +194,17 @@ func main() {
 				continue
 			}
 			log.Printf("generating and certifying P256 key %s", name)
-			_, err := generateAndDeployKeyPair(manifestWrapper.KubernetesNamespace, name)
+			privKey, err := generateAndDeployKeyPair(manifestWrapper.KubernetesNamespace, name)
 			if err != nil {
 				log.Fatalf("%s", err)
 			}
 
-			// TODO(timg) get certificate over the key, insert it here
-			newCertificates[name] = PacketEncryptionCertificate{Certificate: "TODO get certificate"}
+			certificate, err := cert.IssueCertificate(manifestWrapper.KubernetesNamespace, privKey)
```

Comment:
This might have to be updated to manifestWrapper.KubernetesNamespace + the domain